### DISCUSSION TITLE
Working with RHEL 9

### DISCUSSION THREAD

**Discussion by mskarbek** [bootc-dev/bootc#93] [General]
I got it that `bootc` in current form is an experiment and far from production use, but I would like to use it with RHEL 9/Alma Linux 9. For now, it is not possible to build it using the default `rust` package - `rust-1.62.1-1.el9.x86_64`. Rust 1.63 is required by one of the `bootc` dependencies. Not a huge problem, `c9s` package works "fine" and RHEL 9.2 will bring Rust 1.66.
The question is, will `bootc`always will be "problematic" on RHEL 9 bringing the latest stuff from Stream 9?
EDIT: To clarifies, I don't expect any declaration of support for RHEL 9 at any stage. I'm just asking how viable in the future will be testing `bootc` on RHEL 9. I get it that there could be legitimate reasons for not caring at all about RHEL currently.

**Comment by cgwalters** [comment]
Building on CentOS Stream 9 is a current target of this project, and that works today.

> and RHEL 9.2 will bring Rust 1.66.

Right, there may be a period of time where git main for this project builds on C9S but not RHEL 9.x, but at some point a new 9.x will be released for which this project will work.
